mne/time_frequency/tfr.py

```diff
@@ -762,6 +762,14 @@ class _BaseTFR(ContainsMixin, UpdateChannelsMixin, SizeMixin):
     """Base TFR class."""
 
     @property
+    def _data(self):
```

Comment:
It's not meant for public use, it's basically just a wrapper to unify many internal use cases (i.e., the other changes in this PR)